block.go

```diff
@@ -285,6 +284,11 @@ type Block struct {
 	tombstones TombstoneReader
 
 	logger log.Logger
+
+	numBytesChunks    int64
```

Comment:
`uint64` and separate struct maybe for those? Not a blocker.

Reply:
all Size,Write interfaces I have seen so far use int or int64 so I prefer to keep it as is.
Also for `db.opts.MaxBytes` we also use `int64` where `-1` means that the size retention is disabled.

Reply:
Ok, fine by me then

Reply:
Thanks for explaining